app/dashboard/views.py

```diff
@@ -915,25 +915,33 @@ def projects_fetch(request):
     page = clean(request.GET.get('page', 1))
     hackathon_id = clean(request.GET.get('hackathon', ''))
 
-    try:
-        hackathon_event = HackathonEvent.objects.get(id=hackathon_id)
-    except HackathonEvent.DoesNotExist:
-        hackathon_event = HackathonEvent.objects.last()
+    if hackathon_id:
+        try:
+            hackathon_event = HackathonEvent.objects.get(id=hackathon_id)
+        except HackathonEvent.DoesNotExist:
+            hackathon_event = HackathonEvent.objects.last()
```

Comment:
Is this a fallback for some other page that users this endpoint?

Reply:
I believe it’s a fallback for other requests